metagraph/src/cli/align.cpp

```diff
@@ -500,15 +501,18 @@ int align_to_graph(Config *config) {
                     }
 
                     thread_pool.enqueue(process_batch,
-                                        SeqBatch(last_mv_it, std::make_move_iterator(it)),
-                                        mbatch_size);
+                                        SeqBatch(last_mv_it, std::make_move_iterator(it)));
                     ++num_minibatches;
                 }
 
                 logger->trace("Num minibatches: {}, minibatch size: {} KB",
                               num_minibatches, mbatch_size / 1e3);
             } else {
-                thread_pool.enqueue(process_batch, std::move(seq_batch), batch_size);
+                if (it != end) {
+                    thread_pool.enqueue(process_batch, std::move(seq_batch));
+                } else {
+                    process_batch(std::move(seq_batch));
+                }
```

Comment:
why not pass it to the pool as before?

Reply:
IIRC I was trying to debug something a while back. I don't need this anymore